systemtests/tests/py2plug-fd-local-fileset-basic/test-cleanup

```diff
@@ -0,0 +1 @@
+# this file is intentionally empty
```

Comment:
is that correct? No test setup and no teardown?

Reply:
Yes, this is correct. The mutiple tests machinery expects a test setup and cleanup, but the two tests that we now have here both do the setup and cleanup themselves.